src/main/java/ma/hibernate/dao/PhoneDaoImpl.java

```diff
@@ -12,11 +20,43 @@ public PhoneDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Phone create(Phone phone) {
-        return null;
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
```

Comment:
use factory from AbstractDao

Reply:
it's ok? 
![image](https://user-images.githubusercontent.com/61744468/142174721-00df0945-0d1f-46d9-afeb-3d579a60ec74.png)